EMRM.py

```diff
@@ -151,12 +182,28 @@ def get_delta_x(self, word, lpos):
         return(delta_x)
         
 class OneFixation:
+    """ Class of a fixation dwelling at a location (in terms of characters) for a period of time.
+
+    Attributes:
+        lpos: A number indicating landing position in terms of characters.
+        fix_dur: A number indicating fixation duration.
+    """
     def __init__(self, lpos, fix_dur):
         self.lpos = lpos
         self.fix_dur = fix_dur
 
 class OneTrial:
+    """ Class of a trial of identifying a word.
+
+    Attributes:
+        reader: A OneVirtualReader with a vocabulary and some visual properties.
+        word: A string indicating the true word to be identified.
+        x: A vector indicating posterior distribution log-odds.
+        elapsed_time: An integer indicating time steps elapsed.
+        fix_loc: A number indicating current fixation location.
```

Comment:
should `lpos` used elsewhere (e.g., in the OneFixation class) also be `fix_loc`?

Reply:
replaced all `lpos` with `fix_loc`